app/components/UserLookup.tsx

```diff
@@ -0,0 +1,180 @@
+import { useFetcher } from '@remix-run/react'
+import type { action } from 'app/routes/api.users'
+import classnames from 'classnames'
+import type { UseComboboxProps, UseComboboxStateChange } from 'downshift'
+import { useCombobox } from 'downshift'
+import { debounce } from 'lodash'
+import {
+  forwardRef,
+  useCallback,
+  useEffect,
+  useImperativeHandle,
+  useState,
+} from 'react'
+
+import { formatAuthor } from '~/routes/circulars/circulars.lib'
+
+import loaderImage from 'nasawds/src/img/loader.gif'
+
+export interface UserLookup {
+  sub?: string
+  email: string
+  name?: string
+  affiliation?: string
+}
+
+interface UserComboBoxProps
+  extends Omit<
+    UseComboboxProps<UserLookup>,
+    'items' | 'onInputValueChange' | 'itemToString'
+  > {
+  disabled?: boolean
+  className?: string
+  group?: string
+  onChange?: () => void
+}
+
+interface UserComboBoxHandle {
+  reset: () => void
+}
+
+const UserLookupComboBox = forwardRef<UserComboBoxHandle, UserComboBoxProps>(
+  ({ disabled, className, group, onChange, ...props }, ref) => {
```

Comment:
The following will give the React component a display name:

```suggestion
  function UserLookupComboBox({ disabled, className, group, onChange, ...props }, ref) {
```